ppd/raster-interpret.c

```diff
@@ -514,6 +514,17 @@ ppdRasterMatchPPDSize(
   int		i = 0;			/* Loop variable */
   char  	pageSizeRequested[64];  /* Requested PageSize */
 
+  if (!header)
+  {
+    _ppdRasterAddError("Page header cannot be NULL!\n");
+    return (-1);
+  }
+  if (!ppd)
+  {
+    _ppdRasterAddError("PPD file not found!\n");
```

Comment:
Note that here we do not have a case of a file not being found but PPD info not being supplied, better use "PPD info not supplied!\n" here.